src/pip/_internal/commands/debug.py

```diff
@@ -87,7 +88,7 @@ def get_vendor_version_from_module(module_name):
     if not version:
         # Try to find version in debundled module info
         pkg_set = pkg_resources.WorkingSet(
-            [os.path.dirname(getattr(module, '__file__'))]
+            [os.path.dirname(getattr(module, '__file__', None))]
```

Comment:
I think bugbear hints at doing this:

```suggestion
            [os.path.dirname(module.__file__)]
```

Reply:
I did this because a few lines above, something similar was done.

https://github.com/pypa/pip/blob/3c6a25c638cf4e6735de1c22e4d5541701f07eda/src/pip/_internal/commands/debug.py#L85

Should I also fix such instances to use `module.__version__` there, or just fix this instance suggested in the commit?

Reply:
Also, I just made that change and saw that `mypy` for python 2 complains at that line.

```
error: Value of type variable "AnyStr" of "dirname" cannot be "Optional[str]"
                [os.path.dirname(module.__file__)]
```

This is because the type of `module.__file__` is [`Optional[str]`](https://github.com/python/typeshed/blob/master/stdlib/2/types.pyi#L120) in python 2, but [str](https://github.com/python/typeshed/blob/master/stdlib/3/_importlib_modulespec.pyi#L31) in python 3.

I am unsure if this counts as a bug which should be filed against typeshed, or should we just add a `#type: ignore` against this line, which will be removed when we drop python 2 support.

Reply:
Okay,I have gone ahead with the `type: ignore` route for now here in commit https://github.com/pypa/pip/pull/8405/commits/ca7195315d526542aa254a975771d04cfc946533